views/question/show.ejs

```diff
@@ -37,23 +26,24 @@
 
 <br>
 <div class="container">
-    <form>
-        <h1 class="prompt"> <%= record.step %>.  <%= record.prompt %></h1>
-
-        <ul class="answers">
-            <div class="form-check">
-                <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
-                <label class="form-check-label answer" for="exampleRadios1">
-                    <%= record.answer_type%>
-                </label>
-            </div>
-        </ul>
-
-
-        <br>
-        <br>
-        <button class="btn btn-primary float-right" type="submit">Contiue</button>
-    </form>
+    <!-- current question -->
+        <h1 class="prompt"> <%= question.step %>.  <%= question.prompt %></h1>
+        <!-- load available answers -->
+        <% for (let answer of answers) { %>
+            <% if (answer.QuestionId == question.id) { %>
+        <form action="/questions/<%= answer.NextQuestionId %>">
+            <ul class="answers">
+                    <div class="form-check">
+                        <input class="form-check-input" type="radio" >
```

Comment:
Every `type="radio"` tag should have and share the same name attribute. That will ensure that only one will be selectable at any time, which is not happening...

<img width="380" alt="Screen Shot 2020-03-25 at 5 36 51 PM" src="https://user-images.githubusercontent.com/140766/77598514-3d088980-6ebf-11ea-8bbf-7262e842ed63.png">

Reply:
Fixed this issue. Only one option is able to be selected now